cmd/ftl/cmd_secret.go

```diff
@@ -169,3 +171,91 @@ func (s *secretUnsetCmd) Run(ctx context.Context, scmd *secretCmd, adminClient a
 	}
 	return nil
 }
+
+type secretImportCmd struct {
+	Input *string `arg:"" placeholder:"JSON" help:"JSON to import as secrets (read from stdin if omitted). Format: [{\"ref\":\"<module>.<name>\",\"value\": <secret>}, ...]" optional:""`
+}
+
+func (s *secretImportCmd) Help() string {
+	return `
+Imports secrets from a JSON array.
+`
+}
+
+func (s *secretImportCmd) Run(ctx context.Context, scmd *secretCmd, adminClient admin.Client) error {
+	var input []byte
+	var err error
+	if s.Input != nil {
+		input = []byte(*s.Input)
+	} else {
+		input, err = io.ReadAll(os.Stdin)
+		if err != nil {
+			return fmt.Errorf("failed to read config from stdin: %w", err)
+		}
+	}
+	var entries map[string]any
```

Comment:
This should just be a shared type that is used for both marshalling and unmarshalling, probably just `map[cf.Ref]string` (and implement `json.Marshaler`/`Unmarshaler` on `cf.Ref` if it isn't already).

Also the docs suggest it is an array, but this is a map?

Reply:
BTW generally, in Go, you should almost never be JSON marshalling/unmarshalling into `any`, if you know what the intended structure is.

Reply:
This is a case where we would want any though right? Each value is arbitrary json, so I was trying to make the file format like this:
```json
{
   "echo.default_name": "hello world",
   "echo.info": {
          "key1": "value1",
          "key2": "value2"
   }
}
```

Reply:
Ah yes, true. However in that case you should use `json.RawMessage` instead of `any`.